### PR TITLE
feat: POC of detecting user framework

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: src/fpjs-provider.tsx [hunk 1]
```diff
@@ -37,18 +39,26 @@ export function FpjsProvider<TExtended extends boolean>({
   cacheLocation,
   loadOptions,
 }: PropsWithChildren<FpjsProviderOptions>) {
+  const [env, setEnv] = useState<EnvDetails | undefined>()
+
   const clientOptions = useMemo(() => {
+    const integrationInfo = [...(loadOptions.integrationInfo || []), `fingerprintjs-pro-react/${packageInfo.version}`]
+
+    if (env) {
+      integrationInfo.push(env.version ? `${env.name}/${env.version}` : env.name)
```
Comment (ilfa) [inline src/fpjs-provider.tsx:48]: It is not an integration, so the idea was to add it as the third param, like `fingerprintjs-pro-react/${packageInfo.version}/${env.name}`

Comment (TheUnderScorer) [reply src/fpjs-provider.tsx]: Changed!

**Review by ilfa — APPROVED** [review]

**Pull request merged**